examples/playground/index.html

```diff
@@ -160,7 +177,7 @@ <h2 style="margin: 0; font-size: 1.2rem">Help</h2>
         <a-entity id="player" class="camera" camera position="0 1.6 0" look-controls></a-entity>
       </a-entity>
 
-      <a-entity class="environment-settings" gltf-model-plus="#sceneGLB"></a-entity>
+      <a-entity id="gltf-model-plus-entity" class="environment-settings" gltf-model-plus="#sceneGLB"></a-entity>
```

Comment:
You added this id but it doesn't seem to be used.

Reply:
Right, that's a leftover from my first approach.